.github/workflows/build-and-test.yaml

```diff
@@ -12,6 +12,7 @@ jobs:
   build-and-test:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
I know, it is a pretty boring request, however, can we move this change to a different commit?